pallets/providers/src/types.rs

```diff
@@ -75,6 +75,7 @@ pub type BalanceOf<T> =
     <<T as Config>::NativeBalance as Inspect<<T as frame_system::Config>::AccountId>>::Balance;
 
 /// BackupStorageProviderId is the type that represents an ID of a Backup Storage Provider, uniquely linked with an AccountId
+/// TODO: We should make these IDs configurable from the runtime...
```

Comment:
Could be... to what benefit?

Reply:
I thought about this mostly because this pallet is not the only pallet using IDs, although we could also argue that since the `storage-providers` pallet is the handler of IDs, other pallets should have generics and interact with this one through the existing interfaces... both seem valid to me